src/app/page.module.scss

```
@@ -1,156 +1,291 @@
+.heroEmphasis {
+  font-style: italic;
+}
+
+.heroCta {
+  display: inline-flex;
   align-items: center;
-  justify-content: center;
-  font-family: var(--font-geist-sans);
-  background-color: var(--background);
+  gap: $space-3;
+  padding: pxToRem(14) pxToRem(36);
```

Stylelint (17.6.0)

[error] Expected "pxToRem" to be "pxtorem" (function-name-case)

(function-name-case)